docs/script-api.md

```diff
@@ -1064,3 +1042,11 @@ Params: `PlayingSound vanilla_sound`
 - [`ICECAVES`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=COSUBTHEME.ICECAVES) 5
 - [`NEOBABYLON`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=COSUBTHEME.NEOBABYLON) 6
 - [`SUNKENCITY`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=COSUBTHEME.SUNKENCITY) 7
+## Aliases
+We use those to clarify what kind of values can be passed and returned from a function, even if the underlying type is really just an integer or a string. This should help to avoid bugs where one would for example just pass a random integer to a function expecting a callback id.
+### CALLBACK_ID == int;
```

Comment:
should these be in all caps? all caps makes me think these are all constants

Reply:
I'm not sure about it, the enums we have are all caps and I wouldn't have done that so it's out of my comfort zone to determine what other types should or should not be all caps.

Reply:
IMO enums make sense as caps because their values are constants. Here we are defining types so I think these should be PascalCase